regtest.sh

```diff
@@ -240,6 +240,8 @@ make-summary() {
     | sed -e '/TABLE_ROWS/ r rows.html' \
     > $filename
 
+  rm rows.html
```

Comment:
LGTM

On Tue, May 26, 2015 at 8:50 PM, Ilya Mironov <notifications@github.com>
wrote:

> In regtest.sh
> <https://github.com/google/rappor/pull/37#discussion_r31101404>:
>
> > @@ -240,6 +240,8 @@ make-summary() {
> >      | sed -e '/TABLE_ROWS/ r rows.html' \
> >      > $filename
> >
> > +  rm rows.html
>
> I removed the line for now, but still wonder - why do we need the
> rows.html file lying around?
>
It's an intermediate output, and we so far we keep all intermediate
outputs.  We could move them if they're in the way but I think it's nice to
have for debuggability.

Make sure './test.sh all' passes (I think I only checked ./test.sh r-unit)
and then go ahead and submit.  This may catch some lint errors, etc.

thanks,
Andy